### ISSUE TITLE
🐛 [BUG] : Scroll Bottom when new msg incomes

### ISSUE THREAD

**Issue by xuelink** [langx/langx#274]
https://github.com/languageXchange/languageXchange/assets/3280475/1137105f-a438-49f8-8d41-62cbb97e9fc5

**Comment by xuelink** [comment]
```ts
    const bottomPosition = scrollElement.scrollHeight * 0.99;
```

**Comment by xuelink** [comment]
Related w/ #705 
Fixed with #686
